2016-07/july-08.md

```diff
@@ -0,0 +1,63 @@
+## July 8 ([discuss](https://github.com/webpack/meeting-notes/pull/3))
+
+### Attendees
+
+* [Johannes Ewald](http://github.com/jhnns)
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+* [Jesus Rodriguez](https://github.com/foxandxss)
+* [Spencer Stark](https://github.com/spencerstark) 
+
+#### Attendees Notes
+
+* Jesus and Spencer joined us today for the Google hangout. Jesus is working directly with Juho for documentation on webpack/webpack.io and his involvement was pertinent. Spencer is helping Sean take documentation and meeting notes.
+
+### Agenda
+
+#### Medium and Publications
+
+* [The initial post about webpack on medium](https://medium.com/webpack/webpack-its-getting-real-92c60fca1db1#.8nv2qek91) was really well received. Juho said that week’s meetings notes should accompany a publication. Everyone was in agreement. We will add it to the checklist of things we'll make sure is included when we post our weekly publication. Sean suggested to continue to publications on Saturday Morning (CST) and everyone was happy with this. 
+
+* The next article will be on documentation, which is in review now. It will contain a call to action for involvement in the webpack/webpack.io documentation process. We will also want to thank [people for offering their time maintaining Plugins and Loaders](https://github.com/webpack/webpack/issues/2734) which has already had some great feedback.
+
+#### Documentation
+
+* Antwar open source library for documentation. Tobias mentioned previously that we will have a separate page in documentation that will support dynamic content (JavaScript). We will have to wait and see what ideas he is cooking up for documentation tools/utils. 
+
+#### Slack
+
+* Sean mentioned starting a webpack slack org. We tossed the idea around about tying out slack and decided we will pilot the slack org that Sean has. If it is successful, we could potentially open this up to people to generate invites to (like slackr) and join.
```

Comment:
Two comments:

1. Reactiflux left Slack because we were asked to leave – Slack disabled our signups. There had been a few sporadic issues, but at the time we migrated, Slack was working fine.
2. When Reactiflux migrated, Rackt tried to have the official channels for React Router (and maybe Redux as well) be #rackt on Freenode. It didn't work – the channels were dead and all the discussion happened on the new Reactiflux Discord anyway. IRC does everything you need in principle, but tools like Discord or Slack just unify everything in a nice way with e.g. unread tracking, notifications, &c. across platforms. I know you get all that with IRCCloud, but that's not something that just happens – good IRC clients exist, but the median user is going to be stuck and frustrated with a bad IRC client.